awkward-cpp/src/cpu-kernels/awkward_ByteMaskedArray_getitem_nextcarry_outindex.cpp

```diff
@@ -1,39 +1,38 @@
 // BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
 
-#define FILENAME(line) FILENAME_FOR_EXCEPTIONS_C("src/cpu-kernels/awkward_ByteMaskedArray_getitem_nextcarry_outindex.cpp", line)
+#define FILENAME(line)                                          \
+  FILENAME_FOR_EXCEPTIONS_C(                                    \
+      "src/cpu-kernels/"                                        \
+      "awkward_ByteMaskedArray_getitem_nextcarry_outindex.cpp", \
+      line)
```

Comment:
These lines were easier to edit and check in bulk when they were one-liners, but it could be hard to explain to the formatter that only macros are allowed to violate the line width rule.

Before this PR, awkward-cpp/src/cpu-kernels/awkward_ListOffsetArray_reduce_nonlocal_outstartsstops_64.cpp already had a multi-line implementation. That was the only file, so the pattern was already broken (automated scans that assumed that it's one line would have failed to catch that kernel).

So I guess this is fine. At least it's rule-based.